From d9744da6587b0c74194a299bec03215f5c142314 Mon Sep 17 00:00:00 2001
From: Robert Marko <robimarko@gmail.com>
Date: Mon, 31 Jul 2023 13:55:49 +0200
Subject: [PATCH 5/6] SSDK: support kernel 6.6

Identify and configure for kernel 6.6.
This is basically always a moving target for Buildroot.

Signed-off-by: Robert Marko <robimarko@gmail.com>
---
 config            | 4 ++--
 make/linux_opt.mk | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/config b/config
index 1cf78f7b..e90105fd 100755
--- a/config
+++ b/config
@@ -24,8 +24,8 @@ ifeq ($(KVER),$(filter 5.4%,$(KVER)))
 	OS_VER=5_4
 endif
 
-ifeq ($(KVER),$(filter 6.1%,$(KVER)))
-	OS_VER=6_1
+ifeq ($(KVER),$(filter 6.6%,$(KVER)))
+	OS_VER=6_6
 endif
 ifeq ($(KVER), 3.4.0)
 	OS_VER=3_4
diff --git a/make/linux_opt.mk b/make/linux_opt.mk
index 53e535bf..f67c8995 100755
--- a/make/linux_opt.mk
+++ b/make/linux_opt.mk
@@ -442,7 +442,7 @@ ifeq (KSLIB, $(MODULE_TYPE))
       KASAN_SHADOW_SCALE_SHIFT := 3
   endif
 
-  ifeq ($(OS_VER),$(filter 5_4 6_1, $(OS_VER)))
+  ifeq ($(OS_VER),$(filter 5_4 6_6, $(OS_VER)))
       ifeq ($(ARCH), arm64)
           KASAN_OPTION += -DKASAN_SHADOW_SCALE_SHIFT=$(KASAN_SHADOW_SCALE_SHIFT)
        endif
@@ -473,7 +473,7 @@ ifeq (KSLIB, $(MODULE_TYPE))
 
   endif
 
-  ifeq ($(OS_VER),$(filter 4_4 5_4 6_1, $(OS_VER)))
+  ifeq ($(OS_VER),$(filter 4_4 5_4 6_6, $(OS_VER)))
                 MODULE_CFLAG += -DKVER34
                 MODULE_CFLAG += -DKVER32
             MODULE_CFLAG += -DLNX26_22
-- 
2.41.0

